common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -474,6 +474,8 @@ enum class eChannelScanOperationCode : uint8_t {
 #define CHANNEL_SCAN_INVALID_PARAM -1
 #define SCAN_ALL_CHANNELS 0
 
+#define PARAMETER_NOT_CONFIGURED -1
```

Comment:
On the other hand, this file seems to be a c++ compliment file, so constexpr

Reply:
Even through it's most likely the PARAMETER_NOT_CONFIGURED will be used as a `int8_t` I don't want to force it.
And I believe that in this case using macros is better

Reply:
https://github.com/isocpp/CppCoreGuidelines/blob/036324/CppCoreGuidelines.md#es31-dont-use-macros-for-constants-or-functions

Reply:
Changed